aead/src/lib.rs

```diff
@@ -1,7 +1,10 @@
 //! A set of traits designed to support authenticated encryption.
 
-#![no_std]
+#![cfg_attr(not(any(feature = "std", feature = "alloc")), no_std)]
+#![cfg_attr(all(feature = "alloc", not(has_extern_crate_alloc)), feature(alloc))]
```

Comment:
What does this actually help? Old nightlies and betas?

Reply:
Yep

Reply:
Actually, the line above it is wrong, will fix.

Reply:
Personally I don't think that's worth the extra complexity of supporting. If people are running nightly they should be running a recent nightly, and alloc has been stable since 1.36.

Reply:
OK.